hedera-node/src/main/java/com/hedera/services/legacy/services/state/AwareProcessLogic.java

```diff
@@ -193,9 +193,9 @@ private void process(TxnAccessor accessor) {
 			ctx.txnCtx().setStatus(opValidity);
 			return;
 		}
-		logic.doStateTransition();
-
-		ctx.networkCtxManager().finishIncorporating(accessor.getFunction());
+		if (ctx.transitionRunner().tryTransition(accessor)) {
```

Comment:
The `TransitionRunner.tryTransition()` method actually [performs the semantic check](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/txns/TransitionRunner.java) in the preceding lines 184-195; so here we can just accept [`master`'s implementation](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/legacy/services/state/AwareProcessLogic.java#L200) of the `process()` method.

Reply:
Ah good catch. Will fix. Thanks @tinker-michaelj!

Reply:
Done